lightning/src/chain/keysinterface.rs

```diff
@@ -540,6 +540,9 @@ impl InMemorySigner {
 		let witness_script = bitcoin::Address::p2pkh(&::bitcoin::PublicKey{compressed: true, key: remotepubkey}, Network::Testnet).script_pubkey();
 		let sighash = hash_to_message!(&bip143::SigHashCache::new(spend_tx).signature_hash(input_idx, &witness_script, descriptor.output.value, SigHashType::All)[..]);
 		let remotesig = secp_ctx.sign(&sighash, &self.payment_key);
+		let payment_script = bitcoin::Address::p2wpkh(&::bitcoin::PublicKey{compressed: true, key: remotepubkey}, Network::Bitcoin).unwrap().script_pubkey();
```

Comment:
All three of these updated methods document which errors they return, do any of them need updating?

Reply:
It looks like all 3 methods have docs specifically stating under what conditions errors are returned, so I'll need to update all 3.